clients/apple/Shared/LockbookApp.swift

```diff
@@ -4,7 +4,7 @@ import SwiftLockbookCore
 import BackgroundTasks
 
 #if os(macOS)
-import AppKit
```

Comment:
was getting a weird compilation error here, something to do with invalid characters